sympy/sets/sets.py

```diff
@@ -885,6 +886,15 @@ def right_open(self):
         """
         return self._args[3]
 
+    @property
+    def is_empty(self):
+        if self.start < self.end:
+            is_empty = False
```

Comment:
I may be confusing to use the name `is_empty` of the method for a local variable. This could be simply `return False`.